src/components/governance/LiProposal.vue

```diff
@@ -94,35 +94,4 @@ export default {
   }
 }
 </style>
-<style>
-.proposal-status {
-  text-transform: uppercase;
-  font-size: 10px;
-  font-weight: 600;
-  border: 2px solid;
-  padding: 2px 4px;
-  border-radius: 0.25rem;
-  color: var(--grey);
-  border-color: var(--grey);
-}
-
-.proposal-status.rejected {
-  color: var(--danger);
-  border-color: var(--danger);
-}
-
-.proposal-status.depositperiod {
-  color: var(--warning);
-  border-color: var(--warning);
-}
-
-.proposal-status.passed {
-  color: var(--success);
-  border-color: var(--success);
-}
-
-.proposal-status.votingperiod {
-  color: var(--tertiary);
-  border-color: var(--tertiary);
-}
-</style>
+<style scoped src="../../styles/proposal.css"></style>
```

Comment:
i would propose importing the css instead `@import "../../styles/proposal.css";`

Reply:
Yep `<style scoped src="../../styles/proposal.css"></style>` works, I was testing that.